src/alto-ui/Badge/__tests__/Badge.test.js

```diff
@@ -0,0 +1,21 @@
+import React from 'react';
+import { shallow } from 'enzyme';
+
+import Badge from '../Badge';
+
+describe('Badge', () => {
+
+  it('is Badge', () => {
```

Comment:
Change description for more specific e.g. `should render` or `should render without crashing`

Reply:
Description changed to `should render without error`